interceptors/protovalidate/options.go

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) The go-grpc-middleware Authors.
+// Licensed under the Apache License 2.0.
+
+// Copyright 2017 David Ackroyd. All Rights Reserved.
+// See LICENSE for licensing terms.
+
+package protovalidate
+
+import (
+	"golang.org/x/exp/slices"
+	"google.golang.org/protobuf/reflect/protoreflect"
+)
+
+type options struct {
+	ignoreMessages []protoreflect.MessageType
+}
+
+type Option func(*options)
```

Comment:
Could you add a comment to this type please?

Reply:
done